configs/config.json.example

```diff
@@ -32,6 +32,8 @@
     "longer_eggs_first": true,
     "evolve_captured": false,
     "release_pokemon": true,
+    "avoid_circles": false,
+    "max_circle_size": 10,
```

Comment:
What you think about moving these parameter to the `spin_forts` key? They seem out of context here.

```
'spin_forts': {
  'avoid_circles': false,
  max_circle_size: 10
}
```

Makes much more sense for me!